tests/test_e2e_20_flow_manager.py

```diff
@@ -305,3 +424,28 @@ def test_020_flow_another_table(self):
 
     def test_020_flow_another_table_restarting(self):
         self.flow_another_table(restart_kytos=True)
+
+    def flow_table_0(self, restart_kytos=False):
+        """Test if, after adding a flow in another table outside kytos, the
+            flow is removed."""
+        self.net.restart_kytos_clean()
+        time.sleep(5)
+
+        s1 = self.net.net.get('s1')
+        s1.dpctl('add-flow', 'table=0,in_port=1,actions=output:2')
+
+        if restart_kytos:
+            # restart controller keeping configuration
+            self.net.start_controller()
+
+        time.sleep(60)
+
+        s1 = self.net.net.get('s1')
+        flows_s1 = s1.dpctl('dump-flows')
+        assert len(flows_s1.split('\r\n ')) == 1
+
+    def test_020_flow_table_0(self):
+        self.flow_another_table()
```

Comment:
```suggestion
        self.flow_table_0()
```

I think that it should be the new function, no?

Reply:
Arturo has fixed the error on commit c90cfbf